src/test/codegen/unwind-abis/c-unwind-abi-panic-abort.rs

```diff
@@ -19,4 +19,4 @@ pub unsafe extern "C-unwind" fn rust_item_that_can_unwind() {
 // Now, make sure that the LLVM attributes for this functions are correct.  First, make
 // sure that the first item is correctly marked with the `nounwind` attribute:
 //
-// CHECK-NOT: attributes #0 = { {{.*}}nounwind{{.*}} }
+// CHECK: attributes #0 = { {{.*}}nounwind{{.*}} }
```

Comment:
`may_unwind` must not be marked with `nounwind` as otherwise you would get UB when it unwinds instead of an abort. Can you add a check that `may_unwind` gets a different attributes list without the `nounwind` attribute?

Reply:
I just added the `#1` check after @bjorn3's comments -- the original commit doesn't check the attribute of `may_unwind`.  It's not wrong, but the test could be expanded.